README.md

```diff
@@ -28,6 +28,10 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://coveralls.io/ - Display test coverage reports - free for open source
   * https://scrutinizer-ci.com/ - Continuous inspection platform - free for open source
   * https://codecov.io/ - Code coverage tool (SaaS) free for 1 private project and no restrictions for publics repos
+  
+## IDEs
```

Comment:
There's already a IDE section :) https://github.com/ripienaar/free-for-dev#ide-and-code-editing